person.rb

```diff
@@ -1,16 +1,20 @@
 require_relative 'nameable'
-require_relative 'decorator'
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] It would be better if we set the getter and setter for our `rentals`

Reply:
Thanks, @addod19 Great tip.